front/tests/pages/operational-studies/scenario-page.ts

```diff
@@ -32,6 +34,14 @@ class ScenarioPage extends CommonPage {
 
   private readonly scenarioTagsLabel: Locator;
 
+  private readonly conflictsButton: Locator;
+
+  private readonly conflictsList: Locator;
+
+  private readonly trainsButton: Locator;
+
+  private readonly scenarioLeftColumn: Locator;
```

Comment:
Nit: perhaps `trainList` would be a better name, for consistency with `conflictsList` and to be more explicit?

Reply:
Sure